core/src/main/java/io/grpc/ClientInterceptors.java

```diff
@@ -141,7 +141,9 @@ public void cancel() { }
     public void halfClose() { }
 
     @Override
-    public void sendPayload(Object payload) { }
+    public boolean sendPayload(Object payload) {
+      return true;
```

Comment:
I'm not getting what you're saying ... there is no `closed` flag that I can look at here. :/